.buildkite/pipelines/upload_dra_to_s3.yml.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License
+# 2.0 and the following additional limitation. Functionality enabled by the
+# files subject to the Elastic License 2.0 may only be used in production when
+# invoked by an Elasticsearch process with a license key installed that permits
+# use of machine learning features. You may not use this file except in
+# compliance with the Elastic License 2.0 and the foregoing additional
+# limitation.
+
+. .buildkite/scripts/common/base.sh
+
+cat <<EOL
+steps:
+  - label: "Upload DRA artifacts to S3 :s3:"
+    key: "upload_dra_artifacts"
+    depends_on: create_dra_artifacts
+    command:
+      - 'buildkite-agent artifact download "build/distributions/*" .'
+      - 'echo "${RED}This step is disabled until BuildKite migration is complete${NOCOLOR}"'
+      #- "./.buildkite/scripts/steps/upload_dra_to_s3.sh"
```

Comment:
We can keep more of the infrastructure operating by uploading to a different directory - see my other comment.

```suggestion
      - "./.buildkite/scripts/steps/upload_dra_to_s3.sh"
```